### DISCUSSION TITLE
How to delete releases when removed from helmfile?

### DISCUSSION THREAD

**Discussion by mslagle** [helmfile/helmfile#354] [Questions]
While using a large helmfile, we sometimes need to remove charts or change charts.  However, the original chart stays installed in our kubernetes clusters.  After looking through the documentation, I have been unable to find the best way to handle this besides manually performing a helm uninstall against the release no longer in the helmfile.

What is the best way to handle charts being removed from a helmfile to also uninstall the releases that have been deployed?

**Comment by yxxhero** [comment]
@mslagle maybe `installed: false`
